lib/quadquizaminos_web/live/tetris_live.ex

```diff
@@ -955,4 +958,16 @@ defmodule QuadquizaminosWeb.TetrisLive do
   defp block_in_bottom?(x, y, bottom) do
     Map.has_key?(bottom, {x, y})
   end
+
+  defp update_contest(socket) do
+    case socket.assigns[:contest] do
+      nil ->
+        socket
+
+      contest ->
+        if Contests.ended_contest?(contest.id) and is_nil(contest.end_time),
```

Comment:
@TraceyOnim are we fetching the contest when the `contest.end_time` is nil or when we have a value of the end_time?